lib/core/start.js

```diff
@@ -55,7 +55,7 @@ function start (events) {
 
 		var ssl = keystone.get('ssl');
 		var unixSocket = keystone.get('unix socket');
-		var startupMessages = ['KeystoneJS Started:'];
+		var startupMessages = [`KeystoneJS ${keystone.version} started:`];
```

Comment:
should it be?
```diff
- var startupMessages = [`KeystoneJS ${keystone.version} started:`];
+ var startupMessages = [`KeystoneJS v${keystone.version} started:`];
```

I assume `keystone.version` comes from the package.json file directly?

Reply:
Added the 'v'. The version comes from Keystone's `package.json` so in practice looks like:
```
------------------------------------------------
KeystoneJS v4.0.0-rc.0 started:
Keystone Demo is ready on http://0.0.0.0:3000
------------------------------------------------
```